docs/sources/installation/helm/monitor-and-alert/index.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Configure monitoring and alerting
+menuTitle: Configure monitoring and alerting
+description: setup monitoring and alerts for the Helm Chart
+aliases:
+  - /docs/writers-toolkit/latest/templates/task-template
+weight: 100
+keywords:
+  - monitoring 
+  - alert
+  - alerting
+---
+
+# Configure monitoring and alerting
+
+By default this Helm Chart configures meta-monitoring of metrics (service monitoring) and logs (self monitoring).
+
+The `ServiceMonitor` resource works with either the Prometheus Operator or the Grafana Agent Operator, and defines how Loki's metrics should be scraped.
+Scraping this Loki cluster using the scrape config defined in the `SerivceMonitor` resource is required for the included dashboards to work. A `MetricsInstance` can be configured to write the metrics to a remote Prometheus instance such as Grafana Cloud Metrics.
+
+*Self monitoring* is enabled by default. This will deploy a `GrafanaAgent`, `LogsInstance`, and `PodLogs` resource which will instruct the Grafana Agent Operator (installed seperately) on how to scrape this Loki cluster's logs and send them back to itself. Scraping this Loki cluster using the scrape config defined in the `PodLogs` resource is required for the included dashboards to work.
+
+Rules and alerts are automatically deployed.
```

Comment:
Since you have to fix helm-docs anyway and we can't merge just yet, can we change this just a bit?
```suggestion
Rules and alerts are automatically deployed using `PrometheusRule` resources, which requires a running Prometheus operator.
```